public/manifest/production.json

```diff
@@ -18,5 +18,44 @@
       "display_name": "Ethiopia",
       "isd_code": "251"
     }
+  ],
+  "v2": [
+    {
```

Comment:
While we're introducing this, reckon we should prefix this entire payload with a `countries:`?

Reply:
Added in b0978a006